commercetools-models/src/main/java/io/sphere/sdk/orders/commands/OrderFromCartCreateCommand.java

```diff
@@ -18,6 +18,10 @@ static OrderFromCartCreateCommand of(final OrderFromCartDraft draft) {
         return new OrderFromCartCreateCommandImpl(draft);
     }
 
+    static OrderFromCartCreateCommand of(final Cart cart) {
+        return new OrderFromCartCreateCommandImpl(OrderFromCartDraft.of(cart));
+    }
```

Comment:
not needed a Cart implements Versioned

Reply:
I'm replacing with ResourceIdentifier<Cart>